lib/tokens.js

```diff
@@ -342,6 +346,15 @@ export async function resolve({requester, token}) {
     const {tokenBatch} = await _getBatch({id: batchId});
     const {internalId} = tokenBatch;
 
+    if(levelOfAssurance < tokenBatch.minAssuranceForResolution) {
+      throw new BedrockError(
+        'Minimum Level of Assurance not met.',
+        'NotAllowedError', {
+          public: true,
```

Comment:
i think it's helpful to include the details for the expected value and the actual value in cases like this.  So maybe in addition to the existing details:
```
{
  levelOfAssurance,
  minAssuranceForResolution: tokenBatch.minAssuranceForResolution
}
```